### PR TITLE
Added a nb demonstrating efficient gpu utilizations

### PR DESCRIPTION
**Pull request by suppathak**
Related #36 

Exploring:
What happens when we load the models in a lower precision format like INT-8? How is the accuracy, CPU, and memory performance affected? Explain theoretically and show results in a notebook. Touch upon challenges of frameworks like bitsandbytes in production.

### CODE REVIEW

File: notebooks/efficient-gpu-utilization.ipynb [hunk 1]
```diff
@@ -0,0 +1,414 @@
+{
```
Comment (Shreyanand) [inline notebooks/efficient-gpu-utilization.ipynb:1]: <p>can we also print the nvidia-smi output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

**Comment by Shreyanand** [comment]
@suppathak What is pending in this PR?

**Comment by suppathak** [comment]
> @suppathak What is pending in this PR?

Thanks @Shreyanand for reminding me . I will clean it a bit and will conclude it. I will add other topics related to model compression in different notebooks.

File: notebooks/efficient-gpu-utilization.ipynb [hunk 2]
```diff
@@ -0,0 +1,476 @@
...
```
Comment (Shreyanand) [inline notebooks/efficient-gpu-utilization.ipynb:1]: <p>source?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (Shreyanand) [inline notebooks/efficient-gpu-utilization.ipynb:1]: <p>Do you mean <strong>with</strong> quantization? </p><p>Can you provide the percentage change in memory and inference time? it is more intuitive.</p><p>For inference time computation, can you run the cell 10 times and average the time? That makes the result more robust.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

**Review by Shreyanand — CHANGES_REQUESTED** [review]
Requested some minor changes, majorly around adding source of the images and improving results. 
Also, add this notebook in the resource section in the README.

**Comment by suppathak** [comment]
@Shreyanand , Is this good to merge? Lmk if you have any feedback. Thanks.

**Review by Shreyanand — APPROVED** [review]

**Pull request merged**